docs2/site/docs/getting-started/dependency-injection.md

```diff
@@ -110,3 +110,167 @@ public void Install(IWindsorContainer container, IConfigurationStore store)
     );
 }
 ```
+
+# Scoped Services
+
+To use scoped services (e.g. HttpContext Scoped services in ASP.NET Core) you will either need to:
+
+* [Use Schema to get IHttpContextAccessor](#using-schema-to-get-ihttpcontextaccessor)
+* [Use SteroidsDI](#using-steroidsdi)
+* [Use UserContext Scoped IServiceProvider](#using-usercontext-scoped-iserviceprovider)
+
+## Using Schema to get IHttpContextAccessor
+
+You can access scoped services by indirectly using the `IHttpContextAccessor` in order to retrieve the currently scoped `HttpContext` which in turn you can use to retrieve the required scoped services.
+
+E.g. You can have the following extension method:
+
+```csharp
+public static class ContextExtensions
+{
+    public static T GetRequiredScopedService<T>(this IResolveFieldContext context)
+    {
+        var schema = (IServiceProvider) context.Schema;
+
+        var contextAccessor = schema.GetRequiredService<IHttpContextAccessor>();
```

Comment:
```suggestion
        var rootServiceProvider = (IServiceProvider)context.Schema;

        var contextAccessor = rootServiceProvider.GetRequiredService<IHttpContextAccessor>();
```

Reply:
As you are casting the schema to `IServiceProvider`, I suggest that you name the variable `rootServiceProvider`, to more accurately represent the object interface.

Reply:
Makes sense! Thanks! Initially was doing the case on the context accessor line, forgot to look at the variable name after moving it up.